xrviz/dashboard.py

```diff
@@ -139,9 +178,25 @@ def create_indexed_graph(self, *args):
         if sel_data.name in self.data.coords:
                 sel_data = sel_data.to_dataset(name=f'{sel_data.name}_')
 
+        c_low_val = self.kwargs['cmap lower limit']
```

Comment:
I note that both the kwargs block (L160, above) and this colour-limit stuff is repeated.

Reply:
As discussed earlier [here in Set/reset Coords](https://github.com/intake/xrviz/pull/7#discussion_r294266150), var_coords and var_dims are handled separately. For the time being, we should have it like this only(simpler) since projection panel is also using same code. Merging the code of  two cases here will result in more conflict in Projection Panel. As of now it is better to have these separate to understand two cases easily. Code cleanup would be done after all sections are complete.